src/software/simulation/simulator.cpp

```diff
@@ -165,6 +170,22 @@ World Simulator::getWorld() const
     return world;
 }
 
+std::unique_ptr<SSL_WrapperPacket> Simulator::getSslWrapperPacket() const
```

Comment:
über nit, should we do `getSSLWrapperPacket` since its an abbreviation? non-blocking

Reply:
will fix in my next PR